apps/web/configuration/app.config.ts

```diff
@@ -2,7 +2,7 @@ export const appConfiguration = {
   head: {
     viewport: 'minimum-scale=1, initial-scale=1, width=device-width',
     htmlAttrs: {
-      lang: 'en',
+      lang:(process.env.DEFAULTLANGUAGE ?? 'en').toLowerCase(),
```

Comment:
**question**: Does the database save languages in upper case or why do we need this?

Reply:
This is a Failsafe we added. It's not necessarily that we need it, but it's good to have.

Reply:
The locales are already lowercase from backend, we can remove it here.